modules/microvm/virtualization/microvm/netvm.nix

```diff
@@ -139,6 +139,15 @@ let
             ${config.ghaf.security.sshKeys.waypipeSshPublicKeyDir}.options = [ "ro" ];
           };
 
+          users.users."proxy-user-network" = {
+            isNormalUser = true;
+            createHome = false;
+            uid = config.ghaf.users.accounts.loginuid;
```

Comment:
again, why can it not be unique service account?

Reply:
as above